.github/workflows/sdks.yml

```diff
@@ -480,3 +480,95 @@ jobs:
           api-level: 29
           arch: x86_64
           script: ~/test-toolchain.sh
+      # the following steps fetch the cached version of each of the SDKs
+      # and assemble and upload a merged artifact
+      - name: Get cached SDK (x86_64)
```

Comment:
Remove, we already have this from above! 😄

Reply:
Removed in e0500db